guardian/prosemirror-typerighter
# fix: Handle telemetry send errors

## What does this change?

Handle telemetry send errors, which are currently unhandled and clutter our Sentry instrumentation.

## How to test

Introduce a network error when telemetry events are sent, perhaps by turning off the browsers network access by any means and running a document check.

You should see `warn `log messages, but no unhandled errors.

<img width="596" alt="Screenshot 2022-01-28 at 08 49 38" src="https://user-images.githubusercontent.com/7767575/151516220-2e2a8612-aa64-4d6b-939d-2c88222a0697.png">

## How can we measure success?

Fewer unhandled errors cluttering our Sentry logs in our consumer CMS.
